iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -292,9 +295,28 @@ def _adjust_operator_bundle(
                     bundle_metadata, org_enclose_repo_namespace, org_enclose_repo_glue
                 )
         elif customization_type == 'related_bundles':
+            # When recursive_related_bundles is set to True, the call is from a
+            # recureive_related_bundles request. Product teams have customizations
+            # in the order so that when this customization is specified, the bundles
+            # are accessible. For the recursive_related_bundles request, we want the
+            # the images to be in an accessible state. So apply all customizations
+            # that were specified before this one and return.
+            if recursive_related_bundles:
+                log.debug(
+                    'related_bundles customization found with recursive_related_bundles set'
+                    ' to True. No further customizations will be performed for organization'
+                    f' {organization}. Finding recursive_related_bundles'
+                )
+                return labels
```

Comment:
Instead of returning here, we can also just break out of the loop. This will create one return line in the function. Not necessary, but finalization could happen in one place if ever needed.